exercises/two-fer/src/example.erl

```diff
@@ -0,0 +1,13 @@
+-module(example).
+-export([two_fer/1, test_version/0]).
+
+two_fer(Name) ->
+  case Name of
+    "" ->
+      "One for you, one for me.";
+    _ ->
+      "One for " ++ Name ++ ", one for me."
```

Comment:
Please use `lists:flatten/1` and `io_lib:format/2` here.

The current version may cause funny errors when you pass something in that is not a string, while using the above mentioned fuctions will complain in a more human readable way.